libmachine/drivers/rpc/client_driver.go

```diff
@@ -36,7 +35,7 @@ func NewRPCClientDriverFactory() RPCClientDriverFactory {
 }
 
 type RPCClientDriver struct {
-	plugin          localbinary.DriverPlugin
+	plugin          io.Closer
```

Comment:
Commit log is a bit cryptic (short), main reason for doing that is that the only method from localbinary.DriverPlugin which `RPCClientDriver.plugin` uses is `Close()`